### ISSUE TITLE
Material for package README outside of Excel

### ISSUE THREAD

**Issue by jakebeal** [iGEM-Engineering/iGEM-distribution#158]
When we write package documentation, we will ultimately want it to be larger and more complex than we can readily do with just a text box in Excel. 

I think it would be good to the package build process look for an alternate documentation README in a specified location. That can be in Markdown and be included into the automatic build. It would, in turn, be able to include links outward, links to data, images, etc.

**Comment by noahsprent** [comment]
@ethanj801 I think this is what you said that you were going to work on when you get the chance. Feel free to unassign yourself if that isn't the case, or edit the issue to more accurately reflect the technical step that you're working on.

**Comment by ethanj801** [comment]
That is correct. Will hopefully get a chance to work on this before the next meeting.

**Comment by noahsprent** [comment]
Thanks! No rush/pressure, I was just going through issues and sorting things out a bit.
